### PR TITLE
Scene abstraction

### PR DESCRIPTION
**Pull request by razugo**

### CODE REVIEW

File: src/serial/Main.cpp [hunk 1]
```diff
@@ -15,21 +16,20 @@ int main(int argc, char **argv)
...
-
-    auto output = RayTracer::Render(objects);
-    Output::OutputPPM(output);
-    Output::OutputLog(output);
-
-    for(auto object : objects) delete object;
+    // // light
+    // objects.push_back(new Sphere(Vector3( 0.0,     20, -30),     3, Vector3(0.00, 0.00, 0.00), 0, 0.0, Vector3(3,3,3)));
+    
+    Output::OutputPPM(RayTracer::Render(scene));
```
Comment (hyunbumy) [inline src/serial/Main.cpp]: Can you add Output::OutputLog here?

Comment (razugo) [reply src/serial/Main.cpp]: Oh my b i didnt see the new output class

Comment (razugo) [reply src/serial/Main.cpp]: Just added @hyunbumy

**Review by hyunbumy — APPROVED** [review]
LGTM

**Pull request merged**